shoop/admin/templates/shoop/admin/base/_navigation.jinja

```diff
@@ -34,6 +34,20 @@
 {% set front_url=shoop_admin.get_front_url() %}
 {% if front_url %}<a class="nav-btn shop-btn" href="{{ front_url }}" data-toggle="tooltip" data-placement="bottom" title="{% trans %}Visit shop{% endtrans %}"><i class="fa fa-shopping-cart"></i></a>{% endif %}
 
+{% if LANGUAGES|length > 1 %}
+<div class="dropdown">
+    <button class="dropdown-toggle lang-btn" type="button" id="languageMenu" data-toggle="dropdown" aria-expanded="true" data-placement="bottom" title="{% trans %}Language{% endtrans %}">
+        <i class="menu-icon fa fa-language"></i>
+    </button>
+    <ul class="dropdown-menu languages" role="menu" aria-labelledby="languageMenu">
+        {% for code, name in LANGUAGES %}
+        <li><a href="#" class="language{% if LANGUAGE_CODE|upper == code|upper %} active{% endif %}" data-value="{{ code }}" data-set-language-url="{{ url("shoop_admin:set-language") }}">{{ name }}</a></li>
```

Comment:
Hmm -- instead of outputting the URL for every link (seems like a bit of a waste), it would probably be better to add it in [the admin configuration JavaScript object](https://github.com/shoopio/shoop/blob/master/shoop/admin/template_helpers/shoop_admin.py#L71-L85) (which is accessible as `window.ShoopAdminConfig` from JavaScript)?